.github/workflows/build.yml

```diff
@@ -53,12 +53,51 @@ jobs:
           pip install -U numpy
       - name: Build and install
         run: rm -rf .eggs && pip install -e .
-      - name: Run unittests and generate coverage report
+      - name: Run python unittests and generate coverage report
         run: |
           coverage run --branch --source mmdeploy -m pytest -rsE tests
           coverage xml
           coverage report -m
 
+  build_cpu_sdk:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      - name: update
+        run: sudo apt update
+      - name: gcc-multilib
+        run: |
+          sudo apt install gcc-multilib g++-multilib wget libprotobuf-dev protobuf-compiler
+          sudo apt update
+          sudo apt install -y ffmpeg libsm6 libxext6 git ninja-build libglib2.0-0 libxrender-dev libc++1-9 libc++abi1-9
+          sudo add-apt-repository ppa:ignaciovizzo/opencv3-nonfree
+          sudo apt install libopencv-dev lcov wget
+          pkg-config --libs opencv
+      - name: Build and run SDK unit test without backend
+        run: |
+          mkdir -p build && pushd build
+          cmake .. -DCMAKE_CXX_COMPILER=g++-7  -DMMDEPLOY_CODEBASES=all    -DMMDEPLOY_BUILD_SDK=ON     -DMMDEPLOY_BUILD_SDK_PYTHON_API=OFF     -DMMDEPLOY_TARGET_DEVICES=cpu   -DMMDEPLOY_COVERAGE=ON  -DMMDEPLOY_BUILD_TEST=ON
+          make -j2
+          mkdir -p mmdeploy_test_resources/transform
+          wget https://media.githubusercontent.com/media/tpoisonooo/mmdeploy-onnx2ncnn-testdata/main/cat-dog.png
+          cp cat-dog.png mmdeploy_test_resources/transform/
+          ./bin/mmdeploy_tests
```

Comment:
Could fix the ci error for `build_cpu_sdk`:
```
Error: 7-25 09:59:58.818] [mmdeploy] [error] [compose.cpp:24] Unable to find Transform creator: collect. Available transforms: ["CenterCrop", "Collect", "Compose", "DefaultFormatBundle", "ImageToTensor", "LoadImageFromFile", "Normalize", "Pad", "Resize", "ResizeOCR", "TopDownAffine", "TopDownGetBboxCenterScale"]
exception: entry not found (8) @ /home/runner/work/mmdeploy/mmdeploy/csrc/mmdeploy/preprocess/transform/compose.cpp:25
exception: 'keys' is missed in arguments, or it is not an array as expected
Error: 7-25 10:00:01.445] [mmdeploy] [error] [model.cpp:24] 'path/to/not/existing/model' doesn't exist
Error: 7-25 10:00:01.445] [mmdeploy] [error] [model.cpp:15] load model failed. Its file path is 'path/to/not/existing/model'
Error: 7-25 10:00:01.445] [mmdeploy] [error] [model.cpp:65] no ModelImpl can parse buffer
Error: 7-25 10:00:01.445] [mmdeploy] [error] [model.cpp:91] DirectoryModel is already registered
```

Reply:
Ah, the purpose of this test case is to trigger an error.

```bash
  REQUIRE(CreateTransform(compose_cfg, kHost, stream) == nullptr);
```

Reply:
Then, the logging message is a bit confusing. Maybe we could catch the throwed exception and mute the error message if possible.